disorder-core/src/Disorder/QuickCheck.hs

```diff
@@ -0,0 +1,6 @@
+module Disorder.QuickCheck (
```

Comment:
Should this be `Disorder.Core.QuickCheck` and/or exported from `Disorder.Core`? I don't know what the implications of that are though...